examples/parse_file.rs

```diff
@@ -0,0 +1,45 @@
+extern crate bincode;
+extern crate libc;
+extern crate redland_rs;
+#[macro_use]
+extern crate unwrap;
+
+use libc::{c_char, fclose, fdopen, fgets};
+use redland_rs::{KvStorage, Model, Parser, Serializer, Uri};
+use std::ffi::CStr;
+use std::fs::File;
+use std::io::prelude::*;
+use std::os::unix::io::IntoRawFd;
+
+fn main() {
+
+    let file_path = "/home/yoga/rdf/sparql11/sparql11.ttl";
```

Comment:
I guess you'd want to add this file into the repository and use a relative path here?